lib/elixir/lib/enum.ex

```diff
@@ -626,6 +626,74 @@ defmodule Enum do
     end)
   end
 
+  @doc """
+  Counts the enumerable stopping at `limit`.
+
```

Comment:
I'd add a small explanation about why someone would want to use it.

Reply:
added